### ISSUE TITLE
Chrome and PDF's not playing nice

### ISSUE THREAD

**Issue by mike192020** [johnculviner/jquery.fileDownload#122]
since piggy backed this on another I thought that I should just make a new one: https://github.com/johnculviner/jquery.fileDownload/issues/98

I am still having an issue where it would throw the error: Resource interpreted as Document but transferred with MIME type application/pdf:

The url does resolve but when it goes to download, nothing happens not even a failure to download message. I switched the file name in the DB to something other than a PDF and it down loads without fail.

I am in chrome.

**Comment by veenarm** [comment]
I don't believe that's an issue with the JS - refer to : http://stackoverflow.com/questions/6587393/resource-interpreted-as-document-but-transferred-with-mime-type-application-zip

Try adding: Content-Disposition: attachment; filename=???.pdf
to your header response.

I get the same message at work, when using this js and when not, using ancient struts - but the downloads etc all work fine - might be something wrong with your actual download content.
